eng/common/scripts/TypeSpec-Project-Process.ps1

```diff
@@ -29,17 +29,16 @@ function CreateUpdate-TspLocation([System.Object]$tspConfig, [string]$TypeSpecPr
     Write-Host "created service folder $serviceDir"
```

Comment:
This script is under eng/common and cannot be edited in the JS repo. Also this script should no longer be getting used and probably should be deleted. /cc @raych1